config/crd/bases/llamastack.io_llamastackdistributions.yaml

```diff
@@ -169,8 +173,12 @@ spec:
                                         from.  Must be a valid secret key.
                                       type: string
                                     name:
+                                      default: ""
                                       description: |-
                                         Name of the referent.
+                                        This field is effectively required, but due to backwards compatibility is
```

Comment:
Same as above. Just out of curiosity, why was this name ever allowed to be empty ?